build.sbt

```diff
@@ -178,13 +178,4 @@ lazy val beam =
     )
     .enablePlugins(JavaAppPackaging, DockerPlugin, BuildInfoPlugin)
 
-lazy val integrationTests = project
```

Comment:
But we didn't delete the actual module?

Reply:
This commit deletes `modules/integration-tests`, did you mean something else? @chuwy

Reply:
No, that's what I meant - sorry didn't notice somehow.